src/LuaPiGui.cpp

```diff
@@ -476,6 +495,19 @@ static int l_pigui_progress_bar(lua_State *l)
 	return 0;
 }
 
+/*
+ * Function: nextColumn
+ *
+ * Moves drawing position to next column
+ *
+ * > local ui = require 'pigui'
+ * > ui.columns(n_columns, label, has_border)
+ * > ui.text("row 1, column 1")
+ * > ui.nextColumn()
+ * > ui.text("row 1, column 2")
+ * > ui.columns(1)
```

Comment:
See [the NaturalDocs syntax reference](https://www.naturaldocs.org/reference/formatting/index.html#code_and_text_diagrams) for more information on how to format text blocks. Ideally we want to prefer the `--- Lua ---` format instead of prefixing each line with a `> ` symbol.

Reply:
Oh, didn't know that. I don't think we've used it anywhere else, which of course isn't an excuse to not start now.

Reply:
In version 1.x this is not supported. We haven't done it anywhere else, so to be consistent, I'm not doing it here either.